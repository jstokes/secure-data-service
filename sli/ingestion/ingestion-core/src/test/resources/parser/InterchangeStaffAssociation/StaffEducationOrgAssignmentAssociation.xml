<?xml version="1.0" encoding="UTF-8"?>
<InterchangeStaffAssociation xmlns:sli="http://slc-sli/ed-org/0.1"
 xmlns="http://ed-fi.org/0100"
 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 xsi:schemaLocation="http://ed-fi.org/0100 file:/C:/beta/sli/sli/edfi-schema/src/main/resources/edfiXsd-SLI/SLI-Interchange-StaffAssociation.xsd">
    <StaffEducationOrgAssignmentAssociation>
        <StaffReference>
            <StaffIdentity>
                <StaffUniqueStateId>zWoaMYsBN3pH5</StaffUniqueStateId>
            </StaffIdentity>
        </StaffReference>
        <EducationOrganizationReference>
            <EducationalOrgIdentity>
                <StateOrganizationId>f5AQKsDEZTNLR4Lm9qkRHn_Xuo_ceSOpM</StateOrganizationId>
            </EducationalOrgIdentity>
        </EducationOrganizationReference>
        <StaffClassification>School Leader</StaffClassification>
        <PositionTitle>deDxbWQ4lJEfeLSGIFa0cwvczyr-dneEDv</PositionTitle>
        <BeginDate>2014-03-04</BeginDate>
        <EndDate>2013-07-05</EndDate>
    </StaffEducationOrgAssignmentAssociation>
</InterchangeStaffAssociation>
