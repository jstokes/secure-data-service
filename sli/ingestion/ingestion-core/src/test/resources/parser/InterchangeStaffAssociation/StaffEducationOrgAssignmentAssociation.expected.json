{
  "StaffReference":{
    "StaffIdentity":{
      "StaffUniqueStateId":{"_value":"zWoaMYsBN3pH5"}
    }
  },
  "EducationOrganizationReference":{
    "EducationalOrgIdentity":{
      "StateOrganizationId":{"_value":"f5AQKsDEZTNLR4Lm9qkRHn_Xuo_ceSOpM"}
    }
  },
  "StaffClassification":{"_value":"School Leader"},
  "PositionTitle":{"_value":"deDxbWQ4lJEfeLSGIFa0cwvczyr-dneEDv"},
  "BeginDate":{"_value":"2014-03-04"},
  "EndDate":{"_value":"2013-07-05"}
}